@xen-orchestra/lite/src/stories/web-core/checkbox/ui-checkbox.story.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties, settings }"
+    :params="[
+      prop('accent').required().enum('brand', 'success', 'warning', 'danger').preset('brand').widget(),
```

Comment:
You can add the type here:
```suggestion
      prop('accent').type('CheckboxAccent').required().enum('info', 'success', 'warning', 'danger').preset('info').widget(),
```
I've also updated the accent value to match my suggestion in the component.